src/RelayProvider.ts

```diff
@@ -19,6 +19,7 @@ import { AccountKeypair } from './AccountManager';
 import { RelayEvent } from './RelayEvents';
 import { Address } from './types/Aliases';
 import { toBN, toChecksumAddress, toHex } from 'web3-utils';
+import { Transaction } from 'ethereumjs-tx';
```

Comment:
Could you please remove this if not necessary?

Reply:
done